index.html

```diff
@@ -0,0 +1,206 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Document</title>
+  <link rel="stylesheet" href="style.css">
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.4.0/css/all.min.css"
+    integrity="sha512-iecdLmaskl7CVkqkXNQ/ZH/XLlvWZOJyj7Yy7tcenmpD1ypASozpmT/E0iPtmFIB46ZmdtAc9eNBvH0H/ZpiBw=="
+    crossorigin="anonymous" referrerpolicy="no-referrer" />
+</head>
+
+
+<body>
+  <header>
+    <!-- Top bar with social links -->
+    <nav class="top-bar">
+      <ul>
+        <li><a href="https://fb.com/optimuscagedefender"><i class="fa-brands fa-facebook-f"></i></a></li>
+        <li><a href="https://www.linkedin.com/company/optimuscagedefender"><i class="fa-brands fa-twitter"></i></a></li>
+        <li><a href="https://www.oxford.com.com/optimuscagedefender">Language</a></li>
+        <li>
+          <!--My Page-->
+          <a href="https://www.oxford.com.com/optimuscagedefender">My Page</a>
+        </li>
+        <li>
+          <a href="https://www.oxford.com.com/optimuscagedefender">Login</a>
+        </li>
+      </ul>
+    </nav>
+
+    <nav class="navbar">
+      <div class="logo">
+        <img src="./pages/pageAssets/DefenderCage Logo.png" alt="Logo">
+      </div>
```

Comment:
- [ ] As the previous reviewer suggested kindly make sure to wrap the logo in the `<a>` tag so it's clickable and has a link to the home page of your website